samples/graph-proactive-installation/csharp/README.md

```diff
@@ -0,0 +1,79 @@
+# Proactive Installation Sample App
+
+Proactive installation of apps using Graph API and send messages
+
+It shows how to send proactive messages to team (channel) or GroupChat after installing the App.
+
+## Prerequisites
+### Tools
+
+- Microsoft Teams is installed and you have an account
+- [.NET Core SDK](https://dotnet.microsoft.com/download) version 3.1
+- [ngrok](https://ngrok.com/) or equivalent tunnelling solution
+
+## To try this sample
+
+1. Clone the repository
+   ```bash
+    git clone https://github.com/OfficeDev/microsoft-teams-samples.git
+    ```
+2. Run the bot from a terminal or from Visual Studio:
+
+  - Launch Visual Studio
+  - File -> Open -> Project/Solution  
+  - Navigate to `samples/graph-proactive-installation/csharp` folder
+  - Select `ProactiveBot.sln` file
+  - Press `F5` to run the project
+3. Run ngrok - point to port 3978
+   ```bash
+   ngrok http -host-header=rewrite 3978
+4. Go to appsettings.json and add ```MicrosoftAppId``` ,  ```MicrosoftAppPassword```, ```AppCatalogTeamAppId``` information.
+5. Run your app, either from Visual Studio with ```F5``` or using ```dotnet run``` in the appropriate folder.
+6. Update the manifest.json file with ```Microsoft-App-ID```, ```Base Url```   value.
+
+
+### Descriptions MS TeamsApp resource type
+
+- Graph API Authentication  using Application Permissions
+- List of apps in team 
+- Add App to Team
+- List of Apps in Groupchat
+- Add App to Groupchat
+- Add App to the Personal scope of Groupchat members
+
+
+1. Graph API Authentication  using Application Permissions
+    Deploy this app to Azure and give API permissions as Application.For more information (https://docs.microsoft.com/en-us/graph/auth-v2-service)
+2. List of apps in team
+    Retrieve a list of apps installed in the specified team [To get the details we need application Permission].
+    For more information (https://docs.microsoft.com/en-us/graph/auth-v2-service)
```

Comment:
Intenstion of this section is not clear. Could you please check and appropriate changes?